docs/api/buckets.rst

```diff
@@ -3,22 +3,23 @@
 Buckets
 #######
 
-A bucket is the parent object of collections and groups. It can be viewed as
-a namespace where all collections and groups are stored.
+A bucket is the parent object of collections and groups.
```

Comment:
I liked comparing it to a *namespace* - that made things nice and clear, for me.